src/shared/components/functions/perFunction/FluxDocsTooltipContent.tsx

```diff
@@ -20,8 +20,8 @@ const FluxDocsTooltipContent: FC<TooltipProps> = ({
   setHoverdFunction,
 }) => {
   useEffect(() => {
-    setToolTipPopup(true)
-    setHoverdFunction(`${func.package}.${func.name}`)
+    setToolTipPopup && setToolTipPopup(true)
+    setHoverdFunction && setHoverdFunction(`${func.package}.${func.name}`)
```

Comment:
These are optional props, not used in the MinimalistInjectionOption.
Therefore, these really need an existence check (to not error).